test_helpers/fixtures/various_todos.org

```diff
@@ -3,4 +3,13 @@
 * This is not a todo
 * TODO Task with active timestamp and repeater <2020-11-15 Sun +1d>
 * TODO Repeating task
-  SCHEDULED: <2019-11-27 Wed +1d>
+  SCHEDULED: <2019-11-28 Thu +1d>
+* TODO Repeating task with +1d repeater
```

Comment:
The reason why the expectation fails is this line. Adding another headline to `various_todos.org` after the `* TODO Repeating task` leads to the log drawer not to be build. 

Unfortunately, the reason is unclear to me.